Resources/Prototypes/Entities/Objects/Consumable/Food/burger.yml

```
@@ -0,0 +1,308 @@
+  components:
+  - type: Sprite
+    state: ro
+# Tastes like bun, lettuce, sludge.
+
+- type: entity
+  name: xenoburger
+  parent: FoodBurgerBase
+  id: FoodBurgerXeno
+  description: Smells caustic. Tastes like heresy.
+  components:
+  - type: Sprite
+    state: x
+# Tastes like bun, acid.
+
+- type: entity
+  name: bearger
+  parent: FoodBurgerBase
+  id: FoodBurgerBear
+  description: Best served rawr.
+  components:
+  - type: Sprite
+    state: bearger
+
+- type: entity
+  name: clown burger
+  parent: FoodBurgerBase
+  id: FoodBurgerClown
+  description: This tastes funny...
+  components:
+  - type: Sprite
+    state: clown
+
+- type: entity
+  name: mime burger
+  parent: FoodBurgerBase
+  id: FoodBurgerMime
+  description: Its taste defies language.
+  components:
+  - type: Sprite
+    state: mime
+# Tastes like  .
+
+- type: entity
+  name: brain burger
+  parent: FoodBurgerBase
+  id: FoodBurgerBrain
+  description: A strange looking burger. It looks almost sentient.
+  components:
+  - type: Sprite
+    state: brain
+# Tastes like bun, brains.
+
+- type: entity
+  name: ghost burger
+  parent: FoodBurgerBase
+  id: FoodBurgerGhost
+  description: Too Spooky!
+  components:
+  - type: Sprite
+    state: ghost
+# Tastes like bun, ectoplasm.
+
+# Note: I would put a bunch of colored burgers here as listed in the tg .dm but
+# I'd rather wait for a custom burger component.
+
+- type: entity
+  name: spell burger
+  parent: FoodBurgerBase
+  id: FoodBurgerSpell
+  description: This is absolutely Ei Nath.
+  components:
+  - type: Sprite
+    state: spell
+# Tastes like bun, silver.
+
+- type: entity
+  name: big bite burger
+  parent: FoodBurgerBase
+  id: FoodBurgerBig
+  description: Forget the Big Mac. THIS is the future!
+  components:
+  - type: SolutionContainer
+    maxVol: 25
+    contents:
+      reagents:
+      - ReagentId: Nutriment
+        Quantity: 25
+  - type: Sprite
+    state: bigbite
+# Tastes like bun, silver, magic.
+
+- type: entity
+  name: super bite burger
+  parent: FoodBurgerBase
+  id: FoodBurgerSuper
+  description: This is a mountain of a burger. FOOD!
+  components:
+  - type: SolutionContainer
+    maxVol: 40
+    contents:
+      reagents:
+      - ReagentId: Nutriment
+        Quantity: 40
+  - type: Sprite
+    state: superbite
+# Tastes like bun, diabetes.
+
+- type: entity
+  name: five alarm burger
+  parent: FoodBurgerBase
+  id: FoodBurgerFive
+  description: HOT! HOT! HOT!
+  components:
+  - type: Sprite
+    state: fivealarm
+# Tastes like bun, HEAT.
+
+- type: entity
+  name: rat burger
+  parent: FoodBurgerBase
+  id: FoodBurgerRat
+  description: Pretty much what you'd expect...
+  components:
+  - type: Sprite
+    state: rat
+# Tastes like bun, HEAT.
+
+- type: entity
+  name: home run baseball burger
+  parent: FoodBurgerBase
+  id: FoodBurgerBaseball
+  description: It's still warm. The steam coming off of it looks like baseball
```

```suggestion
  description: It's still warm. The steam coming off of it looks like a baseball.
```
or is more missing of the description?